### ISSUE TITLE
Pagination pictures are outdated on patternfly.org

### ISSUE THREAD

**Issue by terezanovotna** [patternfly/patternfly-design#818]
Pagination on patternfly.org is not showing the latest designs. 
https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/pagination#main-content

this is confusing when putting together mocks

this pagination is different than the one designed and approved by @katierik toolbar

@mcarrano

**Comment by mcarrano** [comment]
@terezanovotna can you be more specific about what the differences are or what you find confusing? I don't believe that the content on this page is outdated.

**Comment by karelhala** [comment]
@mcarrano the linked design does not look correct to the things we are constantly told that the top pagination should be collapse one. When @katierik created that awesome toolbar she specifically asked for compact variant of pagination to squeeze it in crowded place of top toolbar[1].

If we look at demos in HTML there is mentioned that top pagination could be compact [2]. I would assume that something similiar would be mentioned on design guidelines - if you have large amount of items in toolbar designers should choose compact variant of pagination.

[1] https://github.com/RedHatInsights/frontend-components/pull/146#issuecomment-534118666
[2] https://www.patternfly.org/v4/documentation/core/demos/table/compact-pagination-demo

**Comment by karelhala** [comment]
Looking at the full documentation it is mentioned there, however it is at the bottom of the screen and not really visible. Can we have 2 screenshots of same page with variant pagination? So designers doesn't have to hunt for the prefered variant.

**Comment by mcarrano** [comment]
Yes, thanks for clarifying this @karelhala . We can update this page to make the compact pagination option more evident.
